src/main/java/de/tum/in/www1/artemis/web/rest/ProgrammingExerciseResourceEndpoints.java

```diff
@@ -12,6 +12,8 @@ public final class ProgrammingExerciseResourceEndpoints {
 
     public static final String IMPORT = PROGRAMMING_EXERCISES + "/import/{sourceExerciseId}";
 
+    public static final String IMPORT_FROM_FILE = PROGRAMMING_EXERCISES + "/import-from-file/{courseId}";
```

Comment:
```suggestion
    public static final String IMPORT_FROM_FILE = "/courses/{courseId}/programming-exercises/import-from-file";
```
I think the proper REST-convention URL would be different.